src/config/server.rs

```diff
@@ -41,6 +44,18 @@ impl Server {
     // TODO: cloning isn't required we can return a ref here
     self.allowed_headers.clone().unwrap_or_default()
   }
+  pub fn get_response_headers(&self) -> HeaderMap {
+    let mut header_map = HeaderMap::new();
+    if let Some(headers) = &self.add_response_headers {
+      for (k, v) in headers {
+        header_map.insert(
+          HeaderName::from_bytes(k.as_bytes()).unwrap(),
+          HeaderValue::from_bytes(v.as_bytes()).unwrap(),
```

Comment:
unwrap could crash the server.

Reply:
Added validation before so that it doesn't happen